Program.cs

```diff
@@ -0,0 +1,113 @@
+﻿using OpenQA.Selenium;
+using Mars.Drivers;
+using Mars.Helpers;
+using System;
+using dotenv.net;
+
+namespace Mars
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // Load environment variables from .env file
+            DotEnv.Load();
```

Comment:
this is not correct you need to implement specflow please see the following for example https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html